imageroot/events/module-removed/00event-validation

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+import json
+import sys
+import agent
+import os
+
+event = json.load(sys.stdin)
+
+if str(event['module']) != os.environ['MAIL_SERVER']:
+    print(agent.SD_DEBUG + "Event ignored: module is different")
+    sys.exit(2)
```

Comment:
Even if event errors are surely not handled, they are logged.

Could you merge the two event steps as one? In code

    if event['module'] == os.environ['MAIL_SERVER']:
         # try-restart 
         
See examples of [agent.run_helper()](https://github.com/search?q=org%3ANethServer+run_helper&type=code)